aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js

```diff
@@ -59,72 +63,110 @@ export const ContainerDrivenLayout = ({ containerRef }) => {
         />
       </PageContent>
       <PageContent>
-        <ContentLayout containerRef={containerRef} />
+        <ContentLayout />
       </PageContent>
     </Page>
   );
 };
 
-ContainerDrivenLayout.propTypes = {
-  containerRef: PropTypes.object,
+const properties = {
+  categories: {
+    label: 'Categories',
+    options: [
+      { label: 'Machine Learning', value: 'Machine learning' },
+      { label: 'Big Data', value: 'Big Data' },
+      { label: 'Data Analytics', value: 'Data analytics' },
+      { label: 'Developer Tools', value: 'Developer tools' },
+      { label: 'Monitoring', value: 'Monitoring' },
+      { label: 'Database', value: 'Database' },
+      { label: 'Data Protection', value: 'Data Protection' },
+    ],
+  },
+  publisher: {
+    label: 'Publishers',
+  },
+  delivery: { label: 'Delivery Methods' },
+  pricing: { label: 'Pricing Models' },
 };
 
-const ContentLayout = ({ containerRef }) => {
-  // containerRef is for demonstration purposes on this site. Most
-  // implementations should likely remove.
-  const breakpoint = useContext(ResponsiveContext);
-  const [filteredApps, setFilteredApps] = useState(allApps);
-  const [filters, setFilters] = useState({});
-
-  // apply filters
-  useEffect(() => {
-    let results = [];
-    let numFilters = 0;
-    Object.values(filters).forEach(filter => (numFilters += filter.length));
+const filterButton = (label, expand, setExpand, property) => {
```

Comment:
This is more of a suggestion re: code patterns we tend to follow, but since this is ultimately rendering a component, I'm wondering if we can instead structure this as:

```
const Filter = ({ label, expand, setExpand, property }) => ...
```

then when we call it, do something like:
```
<Filter label="Category" expand={expandCategory} ... />
```

Reply:
Looking into this, style-wise it feels cleaner and more efficient but I will have to let `propTypes` accept a useState's `expand` and `setExpand`

Reply:
That should be fine, you can set to `boolean` and the appropriate function notation

Reply:
done